lib/model/Nutrient.dart

```diff
@@ -0,0 +1,309 @@
+  vitaminB12,
+
+  /// Vitamin B9
+  vitaminB9,
+
+  /// Vitamin K
+  vitaminK,
+
+  /// Cholesterol
+  cholesterol,
+
+  /// Butyric Acid
+  butyricAcid,
+
+  /// Caproic Acid
+  caproicAcid,
+
+  /// Caprylic Acid
+  caprylicAcid,
+
+  /// Capric Acid
+  capricAcid,
+
+  /// Lauric Acid
+  lauricAcid,
+
+  /// Myristic Acid
+  myristicAcid,
+
+  /// Palmitic Acid
+  palmiticAcid,
+
+  /// Stearic Acid
+  stearicAcid,
+
+  /// Oleic Acid
+  oleicAcid,
+
+  /// Linoleic Acid
+  linoleicAcid,
+
+  /// Docosahexaenoic Acid
+  docosahexaenoicAcid,
+
+  /// Eicosapentaenoic Acid
+  eicosapentaenoicAcid,
+
+  /// Erucic Acid
+  erucicAcid,
+
+  /// Monounsaturated Fats
+  monounsaturatedFat,
+
+  /// Polyunsaturated Fats
+  polyunsaturatedFat,
+
+  /// Alcohol
+  alcohol, // pct
+  /// Pantothenic Acid
+  pantothenicAcid,
+
+  /// Biotin
+  biotin,
+
+  /// Chloride
+  chloride,
+
+  /// Chromium
+  chromium,
+
+  /// Fluoride
+  fluoride,
+
+  /// Iodine
+  iodine,
+
+  /// Manganese
+  manganese,
+
+  /// Molybdenum
+  molybdenum,
+
+  /// Omega 3
+  omega3,
+
+  /// Omega 6
+  omega6,
+
+  /// Trans Fats
+  transFat,
+}
+
+/// Helper class around [Nutrient].
+extension NutrientExtension on Nutrient {
+  /// Tags for nutrients, in the cases when the labels are different.
+  ///
+  /// E.g.:
+  /// * 'saturated-fat' and [saturatedFat] - not the same, we need an entry here
+  /// * 'iron' and [iron] - same label, we don't need an entry here
+  static const Map<Nutrient, String> _specialNutrientTags = <Nutrient, String>{
+    Nutrient.saturatedFat: 'saturated-fat',
+    Nutrient.energyKCal: 'energy-kcal',
+    Nutrient.energyKJ: 'energy-kj',
+    Nutrient.vitaminC: 'vitamin-c',
+    Nutrient.vitaminA: 'vitamin-a',
+    Nutrient.vitaminE: 'vitamin-e',
+    Nutrient.vitaminD: 'vitamin-d',
+    Nutrient.vitaminB1: 'vitamin-b1',
+    Nutrient.vitaminB2: 'vitamin-b2',
+    Nutrient.vitaminPP: 'vitamin-pp',
+    Nutrient.vitaminB6: 'vitamin-b6',
+    Nutrient.vitaminB12: 'vitamin-b12',
+    Nutrient.vitaminB9: 'vitamin-b9',
+    Nutrient.vitaminK: 'vitamin-k',
+    Nutrient.butyricAcid: 'butyric-acid',
+    Nutrient.caproicAcid: 'caproic-acid',
+    Nutrient.caprylicAcid: 'caprylic-acid',
+    Nutrient.capricAcid: 'capric-acid',
+    Nutrient.lauricAcid: 'lauric-acid',
+    Nutrient.myristicAcid: 'myristic-acid',
+    Nutrient.palmiticAcid: 'palmitic-acid',
+    Nutrient.stearicAcid: 'stearic-acid',
+    Nutrient.oleicAcid: 'oleic-acid',
+    Nutrient.linoleicAcid: 'linoleic-acid',
+    Nutrient.docosahexaenoicAcid: 'docosahexaenoic-acid',
+    Nutrient.eicosapentaenoicAcid: 'eicosapentaenoic-acid',
+    Nutrient.erucicAcid: 'erucic-acid',
+    Nutrient.monounsaturatedFat: 'monounsaturated',
+    Nutrient.polyunsaturatedFat: 'polyunsaturated',
+    Nutrient.pantothenicAcid: 'pantothenic-acid',
+    Nutrient.omega3: 'omega-3-fat',
+    Nutrient.omega6: 'omega-6-fat',
+    Nutrient.transFat: 'trans-fat',
+  };
+
+  /// Default units for nutrients.
```

Comment:
On the website, default units depends on countries. e.g. sodium is in grams in EU, and mg in US.

Reply:
> On the website, default units depends on countries. e.g. sodium is in grams in EU, and mg in US.

@stephanegigandet The thing is that:
* regardless of the country, the default unit provided there is a good (good enough?) start
* we do take into consideration the country when we use https://fr.openfoodfacts.org/cgi/nutrients.pl but there are no default units there, where default units would make sense